pelican/__init__.py

```diff
@@ -382,6 +382,16 @@ def get_instance(args):
 
 def main():
     args = parse_arguments()
+    prepare(args)
+
+
+def prepare(args):
```

Comment:
so if some imports pelican from command line from his python app , I can just
    import pelican #then
    pelican.prepare() # prepare in turn runs the run function.